scripts/Muon_Analysis_2.py

```diff
@@ -110,9 +170,11 @@ def saveToProject():
 def loadFromProject(project):
     global muonGUI
     muonGUI = main()
-    muonGUI.loadFromContext(project)
+    muonGUI.dock_widget.loadFromProject(project)
+    # muonGUI.loadFromContext(project)
     return muonGUI
 
+
```

Comment:
does this mean that if I checkout this branch I can then load files into the ADS?

Reply:
I have put in a temporary line to ensure that files are loaded into the ADS. This is purely for testing purposes.